### PR TITLE
Style: Reproduce style from Teo

### LINKED ISSUE #7: Layout code is garbage
I made a good mess of it, needs redone.

This was what I tried to do:

layout

This was before planned features were settled on.

In addition to better code, alternative layout suggestions are welcome.

### PR DESCRIPTION
**Pull request by davismj**
This just provides a template according to the provided spec. It should be used and extended if desired. Pulling this in will delete most of the content on the site and make it look like a literal circus.

Closes #7

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,52 +1,33 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```
Comment (davismj) [inline index.html:1]: https://www.w3.org/TR/html4/

tl;dr "hey guys this spec is 20 years old please don't use it"

Comment (TeoTwawki) [reply index.html:1]: 20 years?
TWENTY?

my god, I'm old. :older_man:

**Review by TeoTwawki — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -1,52 +1,33 @@
...
+			<div><span>server status</span></div>
+			<div><span>player count</span></div>
+		</header>
+		<div>
+			<div>
+				<div>
+					<nav><span>NAV BAR</span></nav>
+					<main><span>new and announcements</span></main>
 				</div>
+				<footer><span>no one needs a footer</span></footer>
```
Comment (TeoTwawki) [inline index.html:28]: foot space is for potential copyright notices if I use somebodies artwork, just fyi

I've merge  this to master as-is and the new "live" branch is whats perma on server so that we can continue to work off master and at times we can live check the results of changes by just branch switching on the server.

Comment (TeoTwawki) [reply index.html:28]: note to self: set up some integration's for automating style notifications and error checks
